### DISCUSSION TITLE
Strange behavior between DirectionalLight and Orthogonal Third Person Follow (and possibly gridmap )

### DISCUSSION THREAD

**Discussion by AcevedoC17** [ramokz/phantom-camera#151] [Question / Help Wanted]
Initially I wasn't going to post this because I had solved another problem I had shared on this forum but for this one I can't seem to figure it out. I can't really put a finger on the steps to reproduce, but I will try my best:

I have a gridmap and each tile has textures.
I set up an Ortho Camera with third person follow and it works fine except for one thing: There is a blinding light that follows the center of the camera. 

I rolled back to a previous  version and implemented it all over again, it seems to be an issue with Orthogonal Camera. If I do the same implementation, but switch to perspective, the lighting is correct. 

If I apply a Y-Offset to the Third Person Follow Phantom Camera, the blinding light disappears, but the lighting overall is wrong, for example, the shadows are gone. 

If I edit the material properties directly, I can disable SpecularMode and the light beam turns into a shadow beam. 

Below I'll share some images

1. Expected Look In Editor and via Phantom Camera Preview
![image](https://github.com/ramokz/phantom-camera/assets/77506847/c275f8d9-47de-4d17-88ba-c51040c1b96e)
![image](https://github.com/ramokz/phantom-camera/assets/77506847/15f7e60f-f535-4df1-b1b1-bb90598a088d)
![image](https://github.com/ramokz/phantom-camera/assets/77506847/f998603c-0791-40d6-916e-b3ac31f451b1)
![image](https://github.com/ramokz/phantom-camera/assets/77506847/77c63256-29f8-42b4-b34e-693257fd7630)
![image](https://github.com/ramokz/phantom-camera/assets/77506847/54307fb9-7b80-4a88-a8c0-bcfc8050134c)

Image 1. Expected Look via Editor
Image 2. Expected Look via Phantom Camera Preview
Image 3. Actual Look in Game (No Y- Offset)
Image 4. Actual Look in Game (Y- Offset of 2) Notice: Some shadows appear depending on the position of the center of the camera
Image 5. Actual Look in Game (Y - Offset of 2) I moved the camera, and shadows disappeared

To summarize:

If I use perspective with third person follow, the game world is presented accurately, if I use ortho, the shadows are gone, and a beam of light appears following the center of the camera, this can be temporarily fixed by applying a Y offset, but if the camera moves down, the beam of light can return. 

An additional test using a camera with no PhantomCameraHost or PhantomCamera attached, the result is expected:
![image](https://github.com/ramokz/phantom-camera/assets/77506847/6b2dee8a-1f91-4ca7-be46-5f97b0513dbe)

I could use some help coming up with more tests or possible solutions.

**Comment by AcevedoC17** [comment]
My latest findings have found out the following:

Changing the spring arm shape to the following shapes consistently remove the light beam:
HeightMap
WorldBoundary
ConcavePolygon

Other shapes when set to large sizes also remove the light beam

However, other aspects of lighting are still wrong.

**Comment by ramokz** [comment]
My guess would be that it has something to do with the distance between the camera and where everything is. Almost like the camera is culling certain things when far enough away. Not sure what causes the bright light to appear, that one is very weird.

I don't feel fully confident this is an addon specific issue, as there shouldn't be anything in that would cause this.
If you copy the various positions/rotations of the camera when the things above change, and if you remove all `PhantomCamera`/`PhantomCameraHost` nodes in the scene, do the problems still occur if you set the camera to those position/rotation values by hand?
